mautic/mautic
# [UI] Increased icon size

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR increases the icons size in tables.






| Before                                 | After
| -------------------------------------- | ---
|   ![image](https://github.com/mautic/mautic/assets/116097999/e46a9242-f626-4cca-9574-46b5a09a6436)                                     | ![image](https://github.com/mautic/mautic/assets/116097999/5e2d593c-db23-4c7d-92d2-6bbb55f7b2a8)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Segments > Create new > Save & Close > Back to the list > see
